doc/basics.rst

```diff
@@ -19,6 +19,17 @@ To register a handler for a hook, just user :py:func:`gossip.register`:
 		... def func():
 		...     print('Called')
 
+:py:func:`gossip.registered` context can be used for registered a handler for a specific code block
```

Comment:
... context can be used to specify a temporarily registered handler around a specific code block